testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
+
+## Thou shalt be consistent within projects
+If a project has a de-facto standard, it should be maintained. The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request.
```

Comment:
Any big changes where we want to change people's mindset and alter our way of working should be communicated in person to people BEFORE any time is put in to writing code. Using a PR to ultimately change things is inevitable, but it should be the last step in the process. I'll reword the sentence slightly.